packages/terra-image/tests/jest/DecorativeImage.test.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ThemeContextProvider from 'terra-theme-context/lib/ThemeContextProvider';
+
+import DecorativeImage from '../../src/variants/_DecorativeImage';
+
+/* Decorative Image component */
+it('should render a default component with empty attribute', () => {
+  const wrapper = shallow(<DecorativeImage src="" alt="default" title="default" />);
+  expect(wrapper).toMatchSnapshot();
+});
+
+it('should render a default component without title attribute', () => {
+  const wrapper = shallow(<DecorativeImage src="" alt="default" title="default" />);
+  expect(wrapper).toMatchSnapshot();
+});
+
+it('should render a default component without aria attributes', () => {
+  const wrapper = shallow(<DecorativeImage src="" alt="default" title="default" aria-label="default label" />);
+  expect(wrapper).toMatchSnapshot();
+});
+
+it('correctly applies the theme context className', () => {
```

Comment:
Do the other themes need a test as well?

Reply:
This test is only to ensure that theme class-names are applied to image. So we would not require to test this with other themes as we are not validating theme style values with this test.